api/environments/tests/test_views.py

```diff
@@ -536,6 +539,53 @@ def test_create_environment_without_required_metadata_returns_400(
     assert "Missing required metadata field" in response.json()["metadata"][0]
 
 
+def test_view_environment_with_staff_track_query_count(
```

Comment:
I'd prefer a slightly more structured test name here:

```suggestion
def test_view_environment_with_staff__query_count_is_expected(
```